src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -291,6 +291,7 @@ internal static class LogMessages
         public const string IDX11023 = "IDX11023: Expecting json reader to be positioned on '{0}', reader was positioned at: '{1}', Reading: '{2}', Position: '{3}', CurrentDepth: '{4}', BytesConsumed: '{5}'.";
         public const string IDX11025 = "IDX11025: Cannot serialize object of type: '{0}' into property: '{1}'.";
         public const string IDX11026 = "IDX11026: Unable to get claim value as a string from claim type:'{0}', value type was:'{1}'. Acceptable types are String, IList<String>, and System.Text.Json.JsonElement.";
+        public const string IDX11027 = "IDX11027: Invalid JsonWebToken handler configuration parameter value provided for {0}";
```

Comment:
For this you can use the formatter for that extra message so the callers don't have to do string concatenation.
```suggestion
        public const string IDX11027 = "IDX11027: Invalid JsonWebToken handler configuration parameter value provided for {0}. {1}";
```